experimental/packages/opentelemetry-sdk-metrics-base/test/util.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes, ValueType } from '@opentelemetry/api-metrics';
+import { InstrumentationLibrary } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
+import { InstrumentDescriptor } from '../src/InstrumentDescriptor';
+import { InstrumentType } from '../src/Instruments';
+
+export const defaultResource = new Resource({
+  resourceKey: 'my-resource',
+});
+
+export const defaultInstrumentDescriptor: InstrumentDescriptor = {
+  name: 'default_metric',
+  description: 'a simple instrument',
+  type: InstrumentType.COUNTER,
+  unit: '1',
+  valueType: ValueType.DOUBLE,
+};
+
+export const defaultInstrumentationLibrary: InstrumentationLibrary = {
+  name: 'default',
+  version: '1.0.0',
+  schemaUrl: 'https://opentelemetry.io/schemas/1.7.0'
+};
+
+export const commonValues: number[] = [1, -1, 1.0, Infinity, -Infinity, NaN];
+export const commonAttributes: Attributes[] = [{}, {1: '1'}, {a: '2'}, new (class Foo{
+a = '1'
+})];
```

Comment:
Not sure if this is allowed as per spec? The new class part - shouldn't that be restricted in Attribute creation?

Reply:
An object that is created with `new` is a "normal" javascript object. That's language-specific so I don't believe there is anything about this in the spec and I don't believe it should be differentiated with plain objects.